.cirrus.yml

```diff
@@ -219,6 +235,14 @@ task:
       env:
         GCLOUD_SERVICE_ACCOUNT_KEY: ENCRYPTED[f12abe60f5045d619ef4c79b83dd1e0722a0b0b13dbea95fbe334e2db7fffbcd841a5a92da8824848b539a19afe0c9fb]
         SHARD: tests
+        DEPOT_TOOLS: "tmp/depot_tools"
```

Comment:
Is there a way to share this between mac and linux? (Maybe have this code in an actual shell script file that gets cold from both shards?)